exchange/xgrid.F90

```diff
@@ -774,7 +773,7 @@ subroutine load_xgrid (xmap, grid, grid_file, grid1_id, grid_id, tile1, tile2, u
         call get_variable_units(fileobj, "xgrid_area", attvalue)
 
         if( trim(attvalue) == 'm2' ) then
-           garea = 4.0*PI*RADIUS*RADIUS;
+           garea = 4.0_r8_kind * real(PI,r8_kind) * real(RADIUS,r8_kind) * real(RADIUS, r8_kind);
```

Comment:
if garea is already 48, then PI and RADIUS do not need real()

Reply:
hmm, i wonder if I should go back to astronomy_mod and make some more changes if thats the case then